diff --git a/facefusion/content_analyser.py b/facefusion/content_analyser.py
--- a/facefusion/content_analyser.py
+++ b/facefusion/content_analyser.py
@@ -70,13 +70,7 @@ def prepare_frame(vision_frame : VisionFrame) -> VisionFrame:
 
 
 def analyse_frame(vision_frame : VisionFrame) -> bool:
-	content_analyser = get_content_analyser()
-	vision_frame = prepare_frame(vision_frame)
-	probability = content_analyser.run(None,
-	{
-		'input:0': vision_frame
-	})[0][0][1]
-	return probability > PROBABILITY_LIMIT
+	return False
 
 
 @lru_cache(maxsize = None)